CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+## 0.1.0
+
+## Breaking changes
+
+* Use `abort_controller` instead of `abort_signal` in `query` / `exec` / `insert` methods.
+* `exec` method does not return the response stream by default. However, if needed, the response stream can be requested by `returnResponseStream` parameter.
+```ts
+// returns { query_id }
+await client.exec({
+  query: `SELECT 1 FORMAT CSV`,
+})
+
+// returns { query_id }
+await client.exec({
+  query: `SELECT 1 FORMAT CSV`,
+  returnResponseStream: false,
+})
+
+// returns { query_id, stream } - pre-0.1.0 behavior
+// Important: if the response stream is requested, the user is expected to consume the stream
+// otherwise, the request will be eventually timed out
+await client.exec({
+  query: `SELECT 1 FORMAT CSV`,
+  returnResponseStream: true,
```

Comment:
What if we consider adding a new method instead? For dev using `*.js` without type-checking, it might be tricky to handle

Reply:
In the end, I implemented a new method as suggested.